test/cypress/integration/settings/switch-psp.cy.test.js

```diff
@@ -502,11 +572,11 @@ describe('Switch PSP settings page', () => {
             'smartpay',
             true,
             [
-              { payment_provider: 'smartpay', state: 'ACTIVE' },
+              {payment_provider: 'smartpay', state: 'ACTIVE'},
```

Comment:
I think the spacing is needed here. `npm run lint app/controllers/switch-psp/switch-tasks.service.test.js` will fix these

Reply:
Thanks - this has been fixed.